mbp-with-mbo/src/main/java/com/epam/cme/mdp3/control/OffHeapSnapshotCycleHandler.java

```diff
@@ -123,7 +123,7 @@ private void clearArray(LongArray array) {
     }
 
     private class LongArray {
-        private BytesStore bytes;
+        private BytesStore<?, ?> bytes;
```

Comment:
Why did you add typification here?

Reply:
Because the interface has them net.openhft.chronicle.bytes.BytesStore<?, ?>